Tests/Objects/IGTestStoryboardCell.m

```diff
@@ -0,0 +1,13 @@
+//
+//  IGTestStoryboardCell.m
+//  IGListKit
+//
+//  Created by Bofei Zhu on 10/20/16.
+//  Copyright © 2016 Instagram. All rights reserved.
+//
+
+#import "IGTestStoryboardCell.h"
+
+@implementation IGTestStoryboardCell
+
```

Comment:
Since it is done in Storyboard, I think we don't need to call something like `[[UILabel alloc] initWithFrame:CGRectMake(0.0, 0.0, width, height)]` to init here.